nixos/modules/services/mail/roundcube.nix

```diff
@@ -96,10 +96,9 @@ in
       description = lib.mdDoc ''
         The maximum attachment size in MB.
 
-        Note: Since roundcube only uses 70% of max upload values configured in php
-        30% is added automatically to [](#opt-services.roundcube.maxAttachmentSize).
+        Note: Since there is some overhead in encoding attachments 37% is added automatically to offset this
```

Comment:
nonblocking:

```suggestion
        ::: {.note}
        Because of overhead in attachment encoding, 37% is automatically added.
        :::
```

Reply:
I'd also like to see https://github.com/roundcube/roundcubemail/issues/7979#issuecomment-808879209 referenced (and why we chose 37%).